File: Scala/Fizzbuzz.scala
// FizzBuzz in scala using cases
// Author: @Moras-del

object Fizzbuzz {
  def main(args: Array[String]): Unit = {
    (1 until 101).map(i=> (i%3, i%5) match {
      case (0, 0) => "FizzBuzz"
      case (0, _) => "Fizz"
      case (_, 0) => "Buzz"
      case _ => i
    }).foreach(println)
  }
}

File: Ruby/fizzbuzz.rb
# FizzBuzz in Ruby
# Author: @tony133

def fizzbuzz(int)
    if int % 3 == 0 && int % 5 == 0
     return "FizzBuzz"
    end
  
    if int % 5 == 0
     return "Buzz"
    end
  
    if int % 3 == 0
     return "Fizz"
    end
end
  